tracer/test/Datadog.Trace.TestHelpers/SpanMetadataV0Rules.cs

```diff
@@ -220,8 +221,24 @@ public static Result IsGrpcV0(this MockSpan span, ISet<string> excludeTags) => R
                 .IsPresent("grpc.method.path")
                 .IsPresent("grpc.method.service")
                 .IsPresent("grpc.status.code")
+                .IsPresent("out.host")
```

Comment:
Weird you add it here, it should be in the metadata file

Reply:
I noted this in the Elasticsearch PR, but this isn't automated yet. I'll update manually for this PR and perform a separate PR to make it automatic 🙂